### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by smortex**
Auto-correct rubocop offenses

### CODE REVIEW

File: modulesync.gemspec [hunk 1]
```diff
@@ -32,4 +32,5 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency 'octokit', '~>4.0'
   spec.add_runtime_dependency 'puppet-blacksmith', '>= 3.0', '< 7'
   spec.add_runtime_dependency 'thor'
+  spec.metadata['rubygems_mfa_required'] = 'true'
```
Comment (smortex) [inline modulesync.gemspec:35]: Rubocop wants this, I do not think it is a bad idea, but maybe GitHub action will be a blocker for this for the release action?

Comment (bastelfreak) [reply modulesync.gemspec:35]: I do not think we currently support 2FA with our setup :( I will try to find some time and investigate

Comment (bastelfreak) [reply modulesync.gemspec:35]: If you disable the cop for now I can merge it if you like

Comment (smortex) [reply modulesync.gemspec:35]: @neomilium did exactly this in #244 so we can close this PR ;-)

**Pull request closed without merging**